src/commands/codepush/patch.ts

```diff
@@ -0,0 +1,101 @@
+import { AppCommand, CommandArgs, CommandResult, help, failure, ErrorCodes, success, getCurrentApp, shortName, longName, required, hasArg, position, name } from "../../util/commandline";
+import { out } from "../../util/interaction";
+import { inspect } from "util";
+import { MobileCenterClient, models, clientRequest, clientCall } from "../../util/apis";
+import * as chalk from "chalk";
+import * as semver from "semver";
+
+const debug = require("debug")("mobile-center-cli:commands:codepush:patch");
+
+@help("Update the metadata for an existing CodePush release")
+export default class PatchCommand extends AppCommand {
+  
+  @help("Specifies one existing deployment name.")
+  @required
+  @name("--existing-deployment-name")
```

Comment:
You don't actually include the `--` in the command name, sorry. This results in the flag `----existing-deployment-name` in the cli which I don't think will actually work.

Remove the leading `--`.